app/src/main/res/layout/item_suggested_edits_contributions.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="?attr/selectableItemBackground"
+        android:orientation="horizontal"
+        android:paddingStart="16dp"
+        android:paddingTop="16dp">
```

Comment:
Not sure but it looks like the view does not have the content align to the center correctly? The description text touches to the bottom and it will make the last item looks weird.

Reply:
yes @cooltey it is actually not aligned to center... the image just lines up on top with the title... per design..

Reply:
If so, maybe add a margin to the bottom of the `recyclerView` would be solution?